src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -1,3 +1,4 @@
+const refreshDropdown      = require('@binary-com/binary-style').selectDropdown;
```

Comment:
do we need to alphabetize import order for binary static?

Reply:
it's based on path